packages/frontend-shared/src/locales/en-US.json

```
@@ -643,5 +643,8 @@
       "invalidSelector": "Invalid",
       "selectorMethodsLabel": "Selector Methods"
     }
+  },
+  "warnings": {
+    "retry": "Retry"
```

I believe that is what the button looks like in the comps.

```suggestion
    "retry": "Try again"
```